### DISCUSSION TITLE
Prisma / Supabase ORM overlaps

### DISCUSSION THREAD

**Discussion by MentalGear** [supabase/supabase#6917] [Questions]
I'm currently wondering what features of Prisma ORM are overlapping with supabase. Maybe someone has some guide on where/when to use one or the other?

In general, it would be great to have some sort of chart to see what features supabase has that are overlapping with other tools, like Prisma.

**Comment by steve-chavez** [comment]
Hm, I don't know much about Prisma features but one key difference is that:

- `supabase-js` is a frontend library: You can use it in your web/mobile applications directly, no extra server required(for hosted Supabase that is).
- Prisma is a backend library: You need to create your endpoints manually in an extra backend server.
